src/utils/data.ts

```diff
@@ -151,3 +151,15 @@ export const processFlowData = [
     bgColor: "#FCEDF3",
   },
 ] satisfies Array<{ [key: string]: string }>;
+
+export const excludedTranscripts = [
+  "edgedevplusplus/2019/mining-firmware-security",
+];
+
+//  manual way to change sources title name that are not good
+
+export const transformSourceName:Record<string,string> = {
+  "magicalcryptoconference":"Magical Crypto Conference",
+  "magicalcryptoconference-2019":"Magical Crypto Conference 2019",
+  "la-bitdevs": "La BitDevs",
```

Comment:
This should be "LA BitDevs" but I didn't submit a suggestion for this because I don't know if any of this JSON or tagging is case sensitive. I don't want to break anything.